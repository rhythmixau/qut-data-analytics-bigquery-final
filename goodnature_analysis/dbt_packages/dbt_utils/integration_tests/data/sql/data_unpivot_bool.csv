version https://git-lfs.github.com/spec/v1
oid sha256:9c66bd10d4fa34a8f3b81a14dc27aba023d1e865d0d00e952147551be2389501
size 149
