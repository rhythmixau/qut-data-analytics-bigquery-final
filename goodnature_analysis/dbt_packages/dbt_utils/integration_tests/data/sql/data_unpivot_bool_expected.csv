version https://git-lfs.github.com/spec/v1
oid sha256:0b731b31c50d795378b159269b1188029dc00449d691519c2fdf177329280e98
size 300
